modules/virt-troubleshooting-cert-rotation-parameters.adoc

```diff
@@ -0,0 +1,33 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-troubleshooting-cert-rotation-parameters_{context}"]
+
+= Troubleshooting
+
+Deleting one or more `certConfig` values will cause them to revert to the default values, unless the default values conflict with the conditions noted above. If the default values conflict with those conditions, you will receive an error.
+
+If you remove the `server.duration` value in the example below, the default value, `24h0m0s` is greater than the value of `ca.duration`, conflicting with the specified conditions.
+
+.Example
+[source,yaml]
+----
+certConfig:
+   ca:
+     duration: 4h0m0s
+     renewBefore: 1h0m0s
+   server:
+     duration: 4h0m0s
+     renewBefore: 4h0m0s
+----
+
+This will result in the following error message:
+
+[source,terminal]
+----
+error: hyperconvergeds.hco.kubevirt.io "kubevirt-hyperconverged" could not be patched: admission webhook "validate-hco.kubevirt.io" denied the request: spec.certConfig: ca.duration is smaller than server.duration
+----
+
+If a conflict in the default values results in an error, the error message will only note the first conflict. Review all `certConfig` values before you proceed.
```

Comment:
I might simplify this to "The error message only mentions the first conflict. Review all `certConfig` values before you proceed." This is partially because it might not always be "a conflict in the default values," IIUC.